### PR TITLE
Outro

### PR DESCRIPTION
**Pull request by orehzzz**
omg guys look at this new outro function lol gg

### CODE REVIEW

File: discord_music_bot/commands/streaming.py [hunk 1]
```diff
@@ -88,6 +88,21 @@ async def queue_spotify(
...
+        track = await wavelink.YouTubeTrack.search(
+            "https://youtu.be/3_-a9nVZYjk", return_first=True
+        )
+        await interaction.response.send_message("It's time to go to sleep")
+        await player.play(track)
+
+        while player.current == track and player.is_playing():
+            await asyncio.sleep(0.5)
+            if player.position >= 58300:
+                print(player.position)
```
Comment (yaroslavpashynskyi) [inline discord_music_bot/commands/streaming.py:103]: Unnecessary print

**Review by yaroslavpashynskyi — CHANGES_REQUESTED** [review]
.

File: discord_music_bot/commands/streaming.py [hunk 2]
```diff
@@ -88,6 +88,20 @@ async def queue_spotify(
 
         await start_playing(interaction, player)
 
+    @client.tree.command(name="outro", description="epic disconnect")
+    async def play_n_leave(interaction: discord.Interaction) -> None:
+        player = await ensure_voice_channel(interaction)
+        track = await wavelink.YouTubeTrack.search(
+            "https://youtu.be/3_-a9nVZYjk", return_first=True
```
Comment (andrewyazura) [inline discord_music_bot/commands/streaming.py:95]: pls move track url to config

File: discord_music_bot/commands/streaming.py [hunk 3]
```diff
@@ -88,6 +88,20 @@ async def queue_spotify(
...
+        )
+        await interaction.response.send_message("It's time to go to sleep")
+        await player.play(track)
+
+        while player.current == track and player.is_playing():
+            await asyncio.sleep(0.5)
+            if player.position >= 58300:
```
Comment (andrewyazura) [inline discord_music_bot/commands/streaming.py:102]: and move this timestamp to config

File: discord_music_bot/commands/streaming.py [hunk 4]
```diff
@@ -88,6 +88,20 @@ async def queue_spotify(
...
+    @client.tree.command(name="outro", description="epic disconnect")
+    async def play_n_leave(interaction: discord.Interaction) -> None:
+        player = await ensure_voice_channel(interaction)
+        track = await wavelink.YouTubeTrack.search(
+            "https://youtu.be/3_-a9nVZYjk", return_first=True
+        )
+        await interaction.response.send_message("It's time to go to sleep")
+        await player.play(track)
+
+        while player.current == track and player.is_playing():
```
Comment (andrewyazura) [inline discord_music_bot/commands/streaming.py:100]: you can add `player.is_playing() or player.is_paused()` to prevent /pause from stopping loop

Comment (orehzzz) [reply discord_music_bot/commands/streaming.py:101]: unfixable

**Review by yaroslavpashynskyi — APPROVED** [review]
LGTM

File: .env.example [hunk 5]
```diff
@@ -8,6 +8,9 @@ WAVELINK_NODE_PASSWORD='password'
 SPOTIFY_CLIENT_ID='client_id'
 SPOTIFY_CLIENT_SECRET='client_secret'
 
+OUTRO_VIDEO_URL='youtube_url'
+OUTRO_VIDEO_TIMESTAMP=12345 #timestamp in milliseconds
```
Comment (andrewyazura) [inline .env.example:12]: rename to `OUTRO_VIDEO_TIMESTAMP_MS`, it's generally a good practice to include units in variables' names and you'll avoid the comment `#timestamp in milliseconds`

**Review by andrewyazura — CHANGES_REQUESTED** [review]

File: discord_music_bot/config.py [hunk 6]
```diff
@@ -20,4 +20,8 @@
         "client_secret": env.str("CLIENT_SECRET"),
     }
 
+with env.prefixed("OUTRO_"):
```
Comment (andrewyazura) [inline discord_music_bot/config.py:23]: you can change `OUTRO_` TO `OUTRO_VIDEO_` since both variables include this prefix

**Review by andrewyazura — CHANGES_REQUESTED** [review]

**Review by andrewyazura — DISMISSED** [review]

**Comment by andrewyazura** [comment]
i'll prepare it for merge

**Review by andrewyazura — APPROVED** [review]

**Pull request merged**